docs/serving/config-ha.md

```diff
@@ -10,7 +10,7 @@ Active/passive HA in Knative is available through leader election, which can be
 
 When using a leader election HA pattern, instances of controllers are already scheduled and running inside the cluster before they are required. These controller instances compete to use a shared resource, known as the leader election lock. The instance of the controller that has access to the leader election lock resource at any given time is referred to as the leader.
 
-HA functionality is enabled by default for all Knative Serving components except for the `autoscaler` component.
```

Comment:
That should remain. By default it is still 1, no?

Reply:
Then I don't understand why we say HA is enabled by default. By default it is 1 replica for all components, no?

Reply:
No i think it's 3

Reply:
No, I don't find any replicas or HPA in https://github.com/knative/serving/blob/master/config/core/deployments/controller.yaml. 3 is the number we set for e2e tests: https://github.com/knative/serving/blob/master/test/e2e-common.sh#L515

Reply:
hm, I thought we did. But then sg.

Reply:
Probably we need to rephrase. That is allowed, but not enabled by default?

Reply:
done

Reply:
Is there any need to include this line at all now that it's available for all components? We already state above:

"Active/passive HA in Knative is available through leader election, which can be enabled after Knative Serving control plane is installed."

Also, if it's not enabled by default for everything, it now seems illogical that we have a section about disabling it immediately after this - shouldn't we have a section about enabling it first?

Reply:
There are two concepts here: leader election and HA. I reworded this line. Please take another look.

Reply:
I think we should also remove the line above:
"Active/passive HA in Knative is available through leader election, which can be enabled after Knative Serving control plane is installed."

This made sense at the time, but doesn't anymore if we're saying leader election is enabled by default.